ext/strscan/strscan.c

```diff
@@ -21,6 +21,7 @@ extern size_t onig_region_memsize(const struct re_registers *regs);
 #endif
 
 #include <stdbool.h>
+#include <ctype.h>
```

Comment:
Could you sort this list in alphabetical order?
```suggestion
#include <ctype.h>
#include <stdbool.h>
```